portal/src/main/webapp/js/src/load-frontend.js

```diff
@@ -18,6 +25,7 @@ window.loadReactApp = function(config) {
                 '<div class="alert alert-warning">' +
                 '<button type="button" class="close" data-dismiss="alert">&times;</button>' +
                 'cbioportal-frontend dev mode, using localhost:3000' +
+                '&nbsp;<a href="#" onclick="clearDevState()">clear</a>' +
```

Comment:
right, now I remember the discussion we had. you're right. I think it would definitely be worth it for that reason to set up some secure way to do it